zetaclient/chains/evm/observer/v2_inbound_tracker.go

```diff
@@ -84,5 +94,5 @@ func (ob *Observer) ProcessInboundTrackerV2(
 		}
 	}
 
-	return fmt.Errorf("no gateway event found in inbound tracker %s", tx.Hash)
+	return errors.Wrapf(ErrEventNotFound, "inbound tracker %s", tx.Hash)
```

Comment:
It would be good to have a unit test for this condition, 
Or an e2e test by adding a random tx hash and verifying the cctx is not mined.

It's not required for this pr, but adding a TODO/ issue would be good.

Reply:
Will add a todo for https://github.com/zeta-chain/node/issues/2669